### PR TITLE
#0: Add create_venv.sh as environment option in installation instructions

### PR DESCRIPTION
**Pull request by tt-rkim**
…

### CODE REVIEW

**Review by TT-billteng — APPROVED** [review]

**Comment by vtangTT** [comment]
Added clang-17 install instructions, please take a look 👍

**Review by vtangTT — APPROVED** [review]

File: INSTALLING.md [hunk 1]
```diff
@@ -37,7 +46,7 @@ sudo apt install software-properties-common=0.99.9.12 build-essential=12.8ubuntu
 wget https://raw.githubusercontent.com/tenstorrent/tt-metal/main/infra/machine_setup/scripts/setup_hugepages.py
 ```
 
-3. Run first setup script.
+2. Run first setup script.
```
Comment (tt-rkim) [inline INSTALLING.md:49]: lel

File: INSTALLING.md [hunk 2]
```diff
@@ -27,6 +27,15 @@ sudo apt update
 sudo apt install software-properties-common=0.99.9.12 build-essential=12.8ubuntu1.1 python3.8-venv=3.8.10-0ubuntu1~20.04.9 libgoogle-glog-dev=0.4.0-1build1 libyaml-cpp-dev=0.6.2-4ubuntu1 libboost-all-dev=1.71.0.0ubuntu2 libsndfile1=1.0.28-7ubuntu0.2 libhwloc-dev graphviz
 ```
 
+Install Clang-17:
```
Comment (tt-rkim) [inline INSTALLING.md:30]: Hmm should we combine these these two shell blocks into one, with a small instruction that says `Install system-level dependencies and Clang-17:`:

I'm concerned people are gonna skip over this block while reading mindlessly.

Comment (vtangTT) [reply INSTALLING.md:30]: Yea, might as well. Changing it rn..

Comment (vtangTT) [reply INSTALLING.md]: changed

File: INSTALLING.md [hunk 3]
```diff
@@ -20,11 +20,16 @@ Note the current compatability matrix:
 
 ---
 
-### Step 2. System-level dependencies
+### Step 2. System-level dependencies and Clang-17
```
Comment (TT-billteng) [inline INSTALLING.md:23]: don't think we need to add clang-17 here

Comment (tt-rkim) [reply INSTALLING.md]: done

**Pull request merged**